modules/mauth-authenticator-apachehttp/src/main/java/com/mdsol/mauth/apache/HttpClientPublicKeyProvider.java

```diff
@@ -47,32 +47,38 @@ public HttpClientPublicKeyProvider(AuthenticatorConfiguration configuration, Sig
   }
 
   private void setupCache() {
-    publicKeyCache =
-      Caffeine.newBuilder()
-        .expireAfterAccess(ttl, TimeUnit.SECONDS)
+    publicKeyCache = Caffeine.newBuilder()
```

Comment:
There's no longer a need to defer creating the cache until the first request, can you please move this to the constructor?

Reply:
updated in https://github.com/mdsol/mauth-jvm-clients/pull/142/commits/a94bc4936cacee46ca89bac115327abf5aac9f04